content/8-1-community-feedback.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 8.1 Druid Community Feedback - An Open Letter 
+date: '2018-10-14'
+published: true
+authors: ['The Dreamgrove']
+showtoc: true
+description: "An open letter to the community about the state of Druids in patch 8.1."
+---
+
+*This is a supplementary post to the [8.1 Druid Megathread](#) on Reddit. You can follow the discussion there.*
+
+# Balance
+
+**Authors**: Nick, Slippykins, Tettles \
+*[Reddit Thread](#)*
+
+**Balance** is in a decent state with decent DPS across all categories, but no considerable strengths. The {{< spell 272871 "Streaking Stars" >}} nerf hit ST DPS hard. Lively Spirit mechanically feels odd: it provides delayed burst DPS and inefficient use of a healing CD. Additionally, it places a contingency on someone else to maximise your own damage. Bottom line: **{{< spell 279647 "Lively Spirit" >}} does not feel good** to play, but it’s strength relative to other traits makes players incentivised to take it.
+
+**Talent builds** have never varied so little. Most content in the game favours burst DPS over sustained: content where you have breaks between DPS (ex: Mythic+, World Quests), content with periodic damage modifiers (ex: Raid), and content where priority targets must be focused down (ex: Mythic+, Raid). Thus, we have a **cookie-cutter talent build** of {{< spell 205636 "Force of Nature" >}}/{{< spell 102560 "Incarnation: Chosen of Elune" >}}/{{< spell 279620 "Twin Moons" >}}/{{< spell 202770 "Fury of Elune" >}} that emphasises burst DPS at the expense of sustained DPS. Not only that, but the clear damage difference between talents on specific rows (ex: {{< spell 279620 "Twin Moons" >}} vs. {{< spell 202354 "Stellar Drift" >}}/{{< spell 202347 "Stellar Flare" >}}) is just too big.
```

Comment:
I don't think this is entirely true, there is a big varience in the talents used in both the L15 talents and L100 talents (maybe with the exception of New Moon, but even that sees some use).

Incarnation is definitely problematic (sadly I don't think blizzard will do anything about it because of the cross spes theme of having a 3 min. 30s Incarnation cd) and if it was changed to something simple like "CA cd is reduced to 2 min." instead, it would allow Starlord to see much more play.

Stellar Flare isn't really a tuning issue, it can't be any stronger without being mandatory for ST dmg. It needs a new mechanic other than just being a generic dot. Buffing the dmg mod on Stellar Drift probably wouldn't hurt, even if it makes us Starfall on two targets that will rarely be a thing we do anyway. Could allow it to potentially pull ahead of Twin Moons with high mob count or short duration AOE.